cfgrammar/src/lib/yacc/grammar.rs

```diff
@@ -88,6 +93,74 @@ pub struct YaccGrammar<StorageT = u32> {
     expectrr: Option<usize>,
 }
 
+#[cfg(feature = "bincode")]
+impl<StorageT, __Context> Decode<__Context> for YaccGrammar<StorageT>
+where
```

Comment:
Ideally this `impl Decode` and the impl of `BorrowDecode` below would just be part of `#[cfg_attr(feature = "bincode", derive(Encode, Decode))]`.

This impl works, it's just not very pretty, so the question is do we want to wait for that or not.

Reply:
I haven't seen any activity/progress towards fixing it, but the bug has been acknowledged.
So I really can't say.  I could look into fixing it myself, but the bincode code base is new to me.

I had for some reason had thought that because we only export serde derives with features enabled that
this could be changed in a non-breaking fashion.

But because of trait bounds, and `_reconstitute` it really does seem to be a breaking change.

Reply:
Then I think this hack (of sorts) is OK. It's probably worth documenting it with something like `// This can be simplified when github.com/xyz/issue/abc is fixed by ...` or similar?

Reply:
Added a comment in [c0ab286](https://github.com/softdevteam/grmtools/pull/530/commits/c0ab286694b76c4972a6fad845ef8b9b29bba3ed)